### DISCUSSION TITLE
What do the statistical time brackets below the execution results mean? Which one is the actual time consumed?

### DISCUSSION THREAD

**Discussion by leezm** [dbeaver/dbeaver#36004] [General]
![image](https://github.com/user-attachments/assets/2956edf3-0148-4080-acbe-6d58ae3d0587)
what's 0.306s fetch mean?

**Comment by TB-HunBoka** [comment]
Dummy person here, but here is my 2 cents:
When you run a query it needs to be sent to the server, the server will process the query (execute). Finally, the server will send you the response, this is fetching the result.

Fetch means "downloading" the data from the server.

The 11s is the total duration. That's contains the execute + fetch time.

**Reply by leezm** [reply]
> Dummy person here, but here is my 2 cents: When you run a query it needs to be sent to the server, the server will process the query (execute). Finally, the server will send you the response, this is fetching the result.
> 
> Fetch means "downloading" the data from the server.
> 
> The 11s is the total duration. That's contains the execute + fetch time.

Thanks a lot!

**Comment by MicBotta** [comment]
Hello, I have the same doubt about the "total" and "fetch" duration.
I used this query as benchmark (PostgreSQL DB):
SELECT *
FROM measures AS m 
LIMIT 500000

I ran the query first against a remote db over the network, and the on a local copy on my machine.
From the remote DB I got:
- 500000 rows fetched - 55 s (0.574s fetch)
while from the local db I got:
-  500000 rows fetched - 1.047s (0.248s fetch)

Now, the select * is trivial, so all the difference is in the network connection between the machines that is quite slow. 
I expected to see this difference in the fetch time, since it should be "the time to fetch data from server to client".

Apparently it is not like this, what is the meaning of this "fetch time" then?
